auditlog_security/models/auditlog_log_line.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Therp B.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import exceptions, models, fields
+
+
+class AuditlogLogLine(models.Model):
+    _inherit = 'auditlog.log.line'
+    _order = "create_date desc"
+
+    user_id = fields.Many2one(
+        'res.users',
+        string="User",
+    )
```

Comment:
Hey, the parent model already has this field:

https://github.com/OCA/server-tools/blob/11.0/auditlog/models/log.py#L15

I think we should use that. If we need the field here also, it can be a related, stored, indexed field.

Reply:
yes fixing immediately. fetching the user_id  of auditlog.log. this works but it is unnecessary,
All the real_user "trick" i used is unnecessary.